lively.ide/js/browser/index.js

```diff
@@ -77,7 +77,10 @@ export default class Browser extends Morph {
     // packageName, moduleName, codeEntity, scroll, textPosition like {row: 0, column: 0}
     var browser = browserOrProps instanceof Browser ?
       browserOrProps : new this(browserOrProps);
-    if (!browser.world()) browser.openInWindow(); 
+    if (!browser.world()) {
+      let window = browser.openInWindow();
+      window.doNotAcceptDropsForThisAndSubmorphs();
```

Comment:
This probably skips a bunch of containers inside the browser who still continue to accept drops. From the top of my head I can think of the wrapper for the top row of buttons as one example of that.